src/views/blocks/parts/details.php

```diff
@@ -17,7 +17,7 @@
 $time_format          = get_option( 'time_format', Tribe__Date_Utils::TIMEFORMAT );
 $time_range_separator = tribe_get_option( 'timeRangeSeparator', ' - ' );
 $show_time_zone       = tribe_get_option( 'tribe_events_timezones_show_zone', false );
-$time_zone_label      = Tribe__Events__Timezones::get_event_timezone_abbr( $event_id );
+$time_zone_label      = Tribe__Events__Timezones::is_mode( 'site' ) ? $time_zone_label = Tribe__Events__Timezones::wp_timezone_abbr( $local_start_time ) : Tribe__Events__Timezones::get_event_timezone_abbr( $event_id );
```

Comment:
you're assigning `$time_zone_label` twice here.

Reply:
I realized this after merging and changed it in this [PR](https://github.com/the-events-calendar/the-events-calendar/pull/3632).